sycl/doc/OptionalDeviceFeatures.md

```diff
@@ -629,11 +629,122 @@ example, if a new device is released before there is a new DPC++ release.  In
 fact, the DPC++ driver supports a command line option which allows the user
```

Comment:
I suggest you remove the following sentences from the first paragraph in this section because the device names are discussed in more depth in the new section you added.

> We expect to define a set of device architecture names that are used consistently in many places (in this
configuration file, in the names of device-specific aspects, as parameters for the `-fsycl-targets` command line option, etc.)  However, we have not yet agreed on these architecture names.

After removing this, the paragraph will state simply:

> The configuration file uses a simple YAML format where each top-level key is a name of a device architecture.  There are sub-keys under each device for the supported aspects, sub-group sizes and AOT compiler ID.  For example:

Reply:
I'll try to reformulate. I think a note that the design requires to have all architecture names consistent everywhere in the compiler is still useful.

Reply:
I agree that it's important to say this, but you already say it in the section "AOT target identification".  There's no need to duplicate it also in the introduction.

Reply:
ok, I see. Will do.